package.json

```diff
@@ -14,6 +14,7 @@
   ],
   "dependencies": {
     "@node-loader/core": "2.0.0",
+    "chalk": "^4.1.2",
```

Comment:
i prefer to use exact version syntax for all the hathora deps (been burned too many times by package breaks)

Reply:
That's what the package-lock.json is for.

Reply:
package-lock is good for local dependencies but it doesn't get published to npm, so it doesn't apply to folks who install hathora